src/main/java/core/net/OnlineWorker.java

```diff
@@ -840,6 +838,14 @@ private static Matchdetails downloadMatchDetails(int matchID, MatchType matchTyp
 		return details;
 	}
 
+	private static boolean downloadOfMatchDetailsFailed(String matchDetails, int matchID) {
```

Comment:
I suggest `matchId` instead of `matchID`.

Reply:
changed